web_client/views/ItemSelectorWidget.js

```diff
@@ -34,7 +34,7 @@ var ItemSelectorWidget = View.extend({
         });
 
         this.$el.html(
-            itemSelectorWidget(this.model.attributes)
+            itemSelectorWidget(this.model.attributes)  // eslint-disable-line backbone/no-view-model-attributes
```

Comment:
`this.model.toJSON()` is the correct way to get a simple object of a Model's attributes.